### PR TITLE
VSL-26: Added test case with 20 signers

### PR DESCRIPTION
**Pull request by KsenijaZivkovic**
Based on branch jackson/VSL-43-events

Added util function that creates an array of signers
Added verification for signers in TestTreasurySetup
Added TestTransferTokensToAccountActionsWith20Signers which has the same flow as TestTransferTokensToAccountActions but has 20 signers

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -150,6 +155,98 @@ func TestTransferTokensToAccountActions(t *testing.T) {
...
+		assert.Equal(otu.T, uint64(0), treasuryBalance)
+
+		// Assert that all funds have been received by the recipient account
+		recipientBalance := otu.GetAccount(RecipientAcct).Balance
+		assert.Equal(otu.T, TransferAmountUInt64+DefaultAccountBalance, recipientBalance)
+	})
+
+	t.Run("Signers should be able to propose a transfer of a non-fungible token out of the Treasury", func(t *testing.T) {
+		// TODO: create collection in one of the signer accounts
+		otu.ProposeNonFungibleTokenTransferAction("treasuryOwner", MaxSigners[0], "account", uint64(0))
```
Comment (mannynotfound) [inline main_test.go:213]: am I reading this right that the recipient is a string of just "account"? Feels like that should fail lol

Comment (jacksonConrad) [reply main_test.go:213]: the recipient in this case is `emulator-account`!  Overflow takes the account name to resolve the address, but you can drop the `emulator-` prefix.

**Comment by jacksonConrad** [comment]
Looks great, approved!  Hold off on merging for now though, please.

**Comment by jacksonConrad** [comment]
closing -- applied changes to `quantstamp-audit/VSL-56_QSP-3`

**Pull request closed without merging**